MiniApp/Classes/core/Display/MiniAppExternalWebViewController.swift

```diff
@@ -66,7 +66,7 @@ extension MiniAppExternalWebViewController: WKNavigationDelegate {
     }
 
     public func webView(_ webView: WKWebView, decidePolicyFor navigationAction: WKNavigationAction, decisionHandler: @escaping (WKNavigationActionPolicy) -> Void) {
-        if navigationAction.navigationType == .linkActivated {
+        if navigationAction.navigationType == .linkActivated || navigationAction.navigationType == .formSubmitted {
```

Comment:
do you think we should have an array of navigation types?
```suggestion
        let navigationTypes: [WKNavigationType] = [.linkActivated, .formSubmitted]
        if navigationTypes.contains(navigationAction.navigationType) {
```